### PR TITLE
DOCS-100 Integrate editorial feedback

### PR DESCRIPTION
**Pull request by pyr0gan**
- All instances of "Web 3" normalized to "Web3"
- Added a function in `generate.js` which reorders the API reference category of the sidebar
- Reordered the columns of the API Reference index table to match
- Fixed the misplaced backtick in the PUT method

Screen Shot 2022-09-01 at 2 27 09 PM

### CODE REVIEW

File: docs/api-reference/index.mdx [hunk 1]
```diff
@@ -9,22 +9,22 @@ slug: "/api-reference"
...
 | BNB Chain  | Mainnet          | Full & Archive |                  ✓                   |                                -                                 |                     -                      |                     -                      |     ✓      |
 | Celo       | Mainnet, Testnet | Archive        |                  ✓                   |                                ✓                                 |                     -                      |                     -                      |     ✓      |
 | Centrifuge | Mainnet          | Full           |                  ✓                   |                                -                                 |                     -                      |                     -                      |     ✓      |
-| Cosmos     | Mainnet          | Full & Archive |                  ✓                   |                                ✓                                 |                     ✓                      |                     -                      |     ✓      |
```
Comment (dgamboa) [inline docs/api-reference/index.mdx:19]: Are these the changes for the transaction search? Can't tell since the table format is wonky here

Comment (pyr0gan) [reply docs/api-reference/index.mdx:19]: Columns were swapped. I agree that the GitHub view here is lacking but if you pull to local and look at it in an IDE it makes more sense (3rd screenshot)

Before:
Screen Shot 2022-09-01 at 2 47 10 PM

After:


Screen Shot 2022-09-01 at 2 47 39 PM

Raw view:

Screen Shot 2022-09-01 at 2 49 06 PM

Comment (pyr0gan) [reply docs/api-reference/index.mdx:19]: This change to the table doesn't touch anything under transaction search, as it wasn't a part of the ticket. I can add another commit here if we don't want to create a separate ticket/PR.

Comment (dgamboa) [reply docs/api-reference/index.mdx:19]: Perfect thanks

**Review by dgamboa — APPROVED** [review]

**Pull request merged**